packages/matrix/scripts/register-test-user.ts

```diff
@@ -1,16 +1,19 @@
 import * as childProcess from 'child_process';
 
+export const adminUsername = 'admin';
+export const adminPassword = 'password';
 
-let username = process.env.USERNAME;
-let password = process.env.PASSWORD;
+let username = process.env.USERNAME || adminUsername;
+let password = process.env.PASSWORD || adminPassword;
 
 (async () => {
   return new Promise<string>((resolve, reject) => {
     childProcess.exec(`docker exec boxel-synapse register_new_matrix_user http://localhost:8008 -c /data/homeserver.yaml -u ${username} -p ${password} --no-admin`,
       (err, stdout) => {
-        if (err && !stdout.includes('User ID already taken')) {
+        if (err) {
```

Comment:
The script was silently failing for me because I already had a user and this was preventing that from being printed. Do you remember why you included this, @FadhlanR?

Reply:
I guess this would make it safe to run twice. The realm server has a test run that looks like it runs this automatically.

I've got a small update I'll commit that should bring this back in while still failing loudly when the user has been created before but the password is different.